packages/manager/modules/pci/src/components/hds/controller.js

```diff
@@ -0,0 +1,8 @@
+import { Environment } from '@ovh-ux/manager-config';
+import { HDS_GET_MORE_INFO } from './hds.constant';
+
+export default class {
+  static getInfoLink() {
```

Comment:
i don't think a static method is needed here, simply :
```js
constructor() {
  this.infoLink = HDS_GET_MORE_INFO[Environment.getUser().ovhSubsidiary];
}
```

Reply:
Fixed.